tests/examples/text/awful/keygrabber/alttab.lua

```diff
@@ -16,7 +16,7 @@ local awful = {keygrabber = require("awful.keygrabber"), --DOC_HIDE
             {{"Mod1", "Shift"}, "Tab", awful.client.focus.history.select_next    },
         },
         -- Note that it is using the key name and not the modifier name.
-        stop_key           = "Alt_L",
+        stop_key           = "Mod1",
```

Comment:
From a quick look at `awful.keygrabber`: This does not work. The keygrabber watches incoming key events and checks if any of them matches what is specified here. `Mod1` is not a key, so it never comes in with a release event.

Reply:
It is handled

```lua
    local converted = generate_conversion_map()[key]

    -- Stop the keygrabber with the `stop_key`
    if (key == self.stop_key or (converted and converted == self.stop_key))
      and event == self.stop_event and self.stop_key then
        self:stop(key, modifiers)
        return false
    end
```